querybook/server/lib/metastore/base_metastore_loader.py

```diff
@@ -139,6 +139,12 @@ def load(self):
                 ]
         self._create_tables_batched(schema_tables)
 
+    def get_latest_partition(self, schema_name, table_name):
+        table, _ = self.get_table_and_columns(schema_name, table_name)
```

Comment:
seems a bit overkill to get the entire table and columns info as partitions are mostly a separate call, can you refactor BaseMetastoreLoader to add a method to get partitions of a table, by default we can return None